lkmpg.tex

```diff
@@ -1701,7 +1701,8 @@ \subsection{Threaded IRQ}
 
 Presence of both handlers is not mandatory.
 If either of them is not needed, pass the \cpp|NULL| instead.
-A \cpp|NULL| top-half handler implicitly means doing nothing but waking up the bottom-half serving thread (for running the bottom-half handler); A \cpp|NULL| bottom-half handler would have the same effect as if \cpp|request_irq()| were used.
+A \cpp|NULL| top-half handler implicitly means doing nothing but waking up the bottom-half serving thread (for running the bottom-half handler);
```

Comment:
Proofread as following:

A \cpp|NULL| top-half handler implies that no action is taken except to wake up the bottom-half serving thread, which runs the bottom-half handler.
Similarly, a \cpp|NULL| bottom-half handler effectively acts as if \cpp|request_irq()| were not used at all.

Reply:
Should be "as if \cpp|request_irq()| were used." instead of "as if \cpp|request_irq()| were not used at all."? I was trying to say that using:

```c
request_threaded_irq(irq, handler, NULL, flags, name, dev);
```

is effectively the same as:

```c
request_threaded_irq(irq, handler, flags, name, dev);
```

Reply:
> Should be "as if \cpp|request_irq()| were used." instead of "as if \cpp|request_irq()| were not used at all."?

Agree. Please go ahead.